app/src/main/java/com/example/bottomnav/ui/dashboard/DashboardFragment.java

```diff
@@ -0,0 +1,44 @@
+package com.example.bottomnav.ui.dashboard;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProvider;
+import com.example.bottomnav.R;
+import com.example.bottomnav.databinding.FragmentDashboardBinding;
+
+public class DashboardFragment extends Fragment {
+
+    private DashboardViewModel dashboardViewModel;
+private FragmentDashboardBinding binding;
+
+    public View onCreateView(@NonNull LayoutInflater inflater,
+            ViewGroup container, Bundle savedInstanceState) {
+        dashboardViewModel =
+                new ViewModelProvider(this).get(DashboardViewModel.class);
+
+    binding = FragmentDashboardBinding.inflate(inflater, container, false);
+    View root = binding.getRoot();
+
+        final TextView textView = binding.textDashboard;
+        dashboardViewModel.getText().observe(getViewLifecycleOwner(), new Observer<String>() {
+            @Override
+            public void onChanged(@Nullable String s) {
+                textView.setText(s);
+            }
+        });
+        return root;
```

Comment:
```suggestion
        return binding.getRoot();
```

We don't use `root` anywhere else, so you can return the thing directly instead of declaring another variable.
(remember to delete the `View root = ...` line above)